examples/sam2_amg_server/README.md

```diff
@@ -41,7 +41,26 @@ The 'ao' mode is a copy of the baseline with modifications to make the code more
 ### 0. Download checkpoints and install requirements
 
 ```
-pip install -r requirements.txt
+# From the top-level "ao" directory
+
+# Create and activate a virtual environment
+python -m venv venv && source venv/bin/activate
+
+# Install requirements for this example
+pip install -r examples/sam2_amg_server/requirements.txt
+
+# Install torch and ao nightlies
+#   (try to uninstall first to be safe, use CUDA 12.1 to be safe)
+pip uninstall torch torchao
+
+pip install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cu121
```

Comment:
I think CUDA 12.4 should be better and available now. The main limitation here is the driver version (see https://docs.nvidia.com/cuda/archive/12.4.0/cuda-toolkit-release-notes/).